pkgs/development/tools/analysis/cvehound/default.nix

```diff
@@ -7,13 +7,13 @@
 
 python3.pkgs.buildPythonApplication rec {
   pname = "cvehound";
-  version = "1.0.9";
+  version = "1.1.0";
 
   src = fetchFromGitHub {
     owner = "evdenis";
     repo = "cvehound";
-    rev = version;
-    hash = "sha256-qwQfpelY1Air3wVQ3RziM/+MNOR3jiKmLpO2w6kXZwM=";
+    rev = "refs/tags/${version}";
```

Comment:
Is there a reason behind this change? It's technically correct but unnecessary.

Reply:
Are there any blockers beside this? Can we merge?

Reply:
Nope, LGTM.